screens/Cart/totalQty.tsx

```diff
@@ -0,0 +1,45 @@
+import { View, Text, StyleSheet } from "react-native";
+import React from "react";
+import { useSelector } from "react-redux";
+
+const totalQty = () => {
```

Comment:
This is a minor thing, but since this is a component, it should start with a capital letter.  EX:  TotalQty.tsx, TotalQty